toolchain/check/testdata/interop/cpp/class/field.carbon

```diff
@@ -63,6 +63,33 @@ fn H(u: Cpp.Union) -> i32 {
   //@dump-sem-ir-end
 }
 
+// --- anon_struct_union.h
+
+struct A {
+  union {
+    struct {
+      short a_0;
+      short b_2;
+      int c_4;
+    };
+    struct {
+      char d_0;
+      char e_1;
```

Comment:
Is it intentional that these two fields not be mapped at the moment - demonstrating that we still get the other fields right despite this? May be worth a comment? (though I realize that comment will get stale once char is supported - so I can see that going either way)